zh_cn.lang

```diff
@@ -355,58 +355,58 @@ effect.xaerominimap.no_minimap_beneficial=小地图不可用
+gui.xaero_box_ingame_waypoint_icon_scale=游戏内路径点图标的缩放。
+gui.xaero_ingame_waypoint_icon_scale=游戏内路径点图标缩放
+gui.xaero_ingame_waypoint_close_scale=近距离游戏内路径点缩放
+gui.xaero_box_ingame_waypoint_close_scale=当你非常靠近它们时，游戏内路径点的缩放。 要更改远处路径点的缩放，请使用 “游戏内路径点图标缩放”、“路径点距离文本缩放”和“路径点名称文本缩放”设置。
+gui.xaero_entity_icon_name_fallback=找不到图标时使用实体名称
 gui.xaero_settings_search=设置搜索
 gui.xaero_settings_search_placeholder=搜索……
-gui.xaero_minimap_view_settings=显示设置
+gui.xaero_minimap_view_settings=视图设置
 gui.xaero_entity_radar_settings=实体雷达设置
 gui.xaero_entity_radar_categories=实体类别设置
 gui.xaero_minimap_block_map_settings=方块地图设置
-gui.xaero_overlay_settings=Overlay Settings
+gui.xaero_overlay_settings=叠加层设置
 gui.xaero_minimap_info_settings=信息设置
 gui.xaero_waypoint_settings=路径点设置
-gui.xaero_minimap_misc_settings=杂项设置
+gui.xaero_minimap_misc_settings=其他设置
 gui.xaero_entity_radar_category=类别：
-gui.xaero_settings_not_found=找不到与您的输入匹配的项！
+gui.xaero_settings_not_found=找不到与你的输入匹配的项！
 gui.xaero_multiple_waypoint_info=多个路径点距离
-gui.xaero_box_multiple_waypoint_info=多个游戏内路径点的距离是否一次性显示。默认情况下，只有距离准星最近的路径点的距离会显示。使用“路径点距离显示”设置以配置哪些路径点的距离会显示。
+gui.xaero_box_multiple_waypoint_info=是否可以同时显示多个游戏内路径点的距离。 默认情况下，仅显示离十字准星最近的路径点的距离。 使用“到路径点的距离”设置来配置可以显示到哪些路径点的距离。
 gui.xaero_while_sneaking=潜行时
 gui.xaero_multiple_waypoints_always=总是
 gui.xaero_narrator_select_nothing=无
 gui.xaero_category_setting_inherit=-
 gui.xaero_category_expanded_options=%s（选择一个选项）
 gui.xaero_entity_category_root=任何
-gui.xaero_entity_category_living=存活
-gui.xaero_entity_category_hostile=敌对
-gui.xaero_entity_category_friendly=友好
-gui.xaero_entity_category_hostile_tamed=驯养
-gui.xaero_entity_category_friendly_tamed=驯养
+gui.xaero_entity_category_living=生物
+gui.xaero_entity_category_hostile=敌对的
+gui.xaero_entity_category_friendly=友好的
+gui.xaero_entity_category_hostile_tamed=驯服的
+gui.xaero_entity_category_friendly_tamed=驯服的
 gui.xaero_entity_category_players=玩家
-gui.xaero_entity_category_same_team=同组
-gui.xaero_entity_category_other_teams=其他组
+gui.xaero_entity_category_same_team=来自相同队伍
+gui.xaero_entity_category_other_teams=来自其他队伍
 gui.xaero_entity_category_items=物品
 gui.xaero_entity_category_other_entities=其他
 gui.xaero_category_settings=设置
 gui.xaero_category_include_list=包含列表
-gui.xaero_box_category_include_list2=“硬包含”中包括的实体顶部的类别或子类别包含的实体类型、
+gui.xaero_box_category_include_list2=要包含在“硬包含”所包含的实体之上的（子）类别中的实体类型列表。
 gui.xaero_category_exclude_list=排除列表
-gui.xaero_box_category_exclude_list2=已经包含在子类中的实体类型中的需要排除的实体类型的列表。可以同时作为“白名单”和“黑名单”，取决于您选择的排除模式。这不会从实体雷达中移除实体，除非您从根类别“任何”中移除。“显示”设置可以将实体从雷达中移除。
+gui.xaero_box_category_exclude_list2=要从已包含在（子）类别中的实体类型中排除的实体类型列表。 根据你选择的排除模式，可以同时用作“白名单”和“黑名单”。 这不会从实体雷达中删除实体，除非你从根“任何”类别中排除。 “显示”设置可以从雷达中排除实体。
 gui.xaero_category_add_subcategory=添加子类别
 gui.xaero_category_hard_include=硬包含
-gui.xaero_box_category_hard_include2=控制哪些实体自动包含在类别或子类别中，无需手动将其加入列表。
-gui.xaero_category_list_add=加入列表……
-gui.xaero_category_exclude_list_mode=排除模式……
+gui.xaero_box_category_hard_include2=控制哪些实体自动包含在（子）类别中，无需手动将它们放入包含列表中。
```

Comment:
For some reason translates back specifically as "subcategories", which is incorrect. Does this mean sub-category, not plural?